lib/mailchimp3.rb

```diff
@@ -5,8 +5,8 @@
 module MailChimp3
   module_function
 
-  def new(*args)
-    Endpoint.new(*args)
+  def new(**kwargs)
```

Comment:
I see that `Endpoint.new` only accepts kwargs, and since we control that code, I don't particularly expect it to change. That being said, do we lose anything by using `...` to forward all args, effectively keeping the same as it was pre Ruby 3 and removing the need to make any changes here in the future? (not a blocker)

Reply:
Good question! Using `...` would work for our purposes, but would be incompatible with Ruby < 2.7: https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/#delegation-ruby-3

I contemplated using it at first but ultimately decided to opt for broader compatibility since this is a public gem. I'm not tied to this approach though – if you (or anyone else) would prefer to use the newer argument delegation syntax, we could always go that route and mark the gem as requiring Ruby 2.7+.

Reply:
Makes perfect sense 👍🏼